AppAuth-JS/built/index.d.ts

```diff
@@ -0,0 +1,18 @@
+export * from './authorization_request';
```

Comment:
This is generated source file. All the files in built/ folder are generated JS file for the Google's lib fork